phpBB/styles/prosilver/theme/content.css

```diff
@@ -491,18 +491,17 @@ blockquote cite > div {
 
 /* Code block */
 .codebox {
-	padding: 3px;
 	border: 1px solid transparent;
 	font-size: 1em;
 	margin: 1em 0 1.2em 0;
-	overflow-x: scroll;
 	word-wrap: normal;
 }
 
 .codebox p {
 	text-transform: uppercase;
 	border-bottom: 1px solid transparent;
-	margin-bottom: 3px;
+	margin-bottom: 0;
+	padding: 3px 3px 0 3px;
```

Comment:
The "CODE: SELECT ALL" didn't have any bottom padding before. Plus, I've tried it with 3px for all and I preferred the way it is now. If you also try it out and think it is better with a 3px bottom padding, then I'll proceed to change it.